Makefile

```diff
@@ -0,0 +1,24 @@
+PROJECT_NAME=$(shell basename "$(PWD)")
+
+tidy:
+	go mod tidy
+
+vet:
+	go vet ./...
+
+test:
+	go test ./... -v
+
+BINARIES_DIRECTORY=bin
+MAIN_FILE=cmd/main.go
+CONFIG_FILE=examples/minimal.toml
+LDFLAGS="-w -s"
+
+clean:
+	rm -rf ${BINARIES_DIRECTORY}
```

Comment:
Can you please fix this for specific path for exact files (without -r flag)?
Recursive rm -rf is practice i would like to avoid

Reply:
Thanks, fixed